### PR TITLE
frontend/bug remove tooltip from table except from studio-url

### PR DESCRIPTION
**Pull request by Fijo-Lurk**
-fix app-id transaltion from app-id to application id

### CODE REVIEW

**Comment by Fijo-Lurk** [comment]
[tabel header change name from app id to Application id](https://trello.com/c/wCVgQlPx/42-tabel-header-change-name-from-app-id-to-application-id)

**Comment by Fijo-Lurk** [comment]
[remove tooltip from table except from Studio url](https://trello.com/c/B0pmkOQx/43-remove-tooltip-from-table-except-from-studio-url)

File: frontend/src/assets/i18n/se.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "editColumn": "Uppdatera",
     "customerId": "Kundnummer",
     "environment": "Miljö",
-    "appId": "App-id",
+    "appId": "Applikation id",
```
Comment (mblomdahl) [inline frontend/src/assets/i18n/se.json:12]:
```suggestion
    "appId": "Applikations-ID",
```

File: frontend/src/assets/i18n/en.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "editColumn": "Edit",
     "customerId": "Customer id",
     "environment": "Environment",
-    "appId": "App id",
+    "appId": "Application id",
```
Comment (mblomdahl) [inline frontend/src/assets/i18n/en.json:12]:
```suggestion
    "appId": "Application ID",
```

File: frontend/src/assets/i18n/en.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "editColumn": "Edit",
     "customerId": "Customer id",
```
Comment (mblomdahl) [inline frontend/src/assets/i18n/en.json:10]:
```suggestion
    "customerId": "Customer ID",
```

File: frontend/src/assets/i18n/se.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "editColumn": "Uppdatera",
     "customerId": "Kundnummer",
```
Comment (mblomdahl) [inline frontend/src/assets/i18n/se.json:10]:
```suggestion
    "customerId": "Kund-ID",
```

**Review by mblomdahl — APPROVED** [review]
Wording corrections! 🥲

**Pull request merged**